Operator-sdk makes syscall, specific to linux making it non-portable to windows.

## Bug Report

**What did you do?**
I tried to compile operator-sdk for windows, got following error - 
 github.com/operator-framework/operator-sdk/pkg/ansible/proxy
pkg\ansible\proxy\kubectl.go:243:13: undefined: syscall.Umask
pkg\ansible\proxy\kubectl.go:245:2: undefined: syscall.Umask
make: *** [install] Error 2

syscall specific to linux and won't work for windows

**Environment**
* operator-sdk version: 0.6.0 

-Can make it generic so it can be compiled on windows as well.

--- a/Gopkg.lock
+++ b/Gopkg.lock
@@ -1529,7 +1529,7 @@
   revision = "0cf8f7e6ed1d2e3d47d02e3b6e559369af24d803"
 
 [[projects]]
-  digest = "1:c4633d96d5a84297fe4b7045060034c6f219d9a18bcd53b88bd560bdf976194f"
+  digest = "1:a2ca0c9095f3fd4160abcb64ebc3d99d3e2439da9c350705db7ab65dc98ee21d"
   name = "k8s.io/kubernetes"
   packages = [
     "pkg/api/legacyscheme",
@@ -1610,6 +1610,7 @@
     "pkg/kubectl/cmd/util/openapi/validation",
     "pkg/kubectl/generated",
     "pkg/kubectl/scheme",
+    "pkg/kubectl/util",
     "pkg/kubectl/util/i18n",
     "pkg/kubectl/util/printers",
     "pkg/kubectl/util/templates",
@@ -1806,6 +1807,7 @@
     "k8s.io/helm/pkg/storage/driver",
     "k8s.io/helm/pkg/tiller",
     "k8s.io/helm/pkg/tiller/environment",
+    "k8s.io/kubernetes/pkg/kubectl/util",
     "sigs.k8s.io/controller-runtime/pkg/cache",
     "sigs.k8s.io/controller-runtime/pkg/client",
     "sigs.k8s.io/controller-runtime/pkg/client/config",

--- a/pkg/ansible/proxy/kubectl.go
+++ b/pkg/ansible/proxy/kubectl.go
@@ -28,13 +28,13 @@ import (
 	"os"
 	"regexp"
 	"strings"
-	"syscall"
 	"time"
 
 	utilnet "k8s.io/apimachinery/pkg/util/net"
 	k8sproxy "k8s.io/apimachinery/pkg/util/proxy"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/transport"
+	"k8s.io/kubernetes/pkg/kubectl/util"
 	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
 )
 
@@ -240,9 +240,9 @@ func (s *server) ListenUnix(path string) (net.Listener, error) {
 		}
 	}
 	// Default to only user accessible socket, caller can open up later if desired
-	oldmask := syscall.Umask(0077)
+	oldmask, _ := util.Umask(0077)
 	l, err := net.Listen("unix", path)
-	syscall.Umask(oldmask)
+	util.Umask(oldmask)
 	return l, err
 }
 

